room/compiler-processing/src/main/java/androidx/room/compiler/processing/javac/JavacFiler.kt

```diff
@@ -22,11 +22,13 @@ import com.squareup.kotlinpoet.FileSpec
 import javax.annotation.processing.ProcessingEnvironment
 
 internal class JavacFiler(val processingEnv: ProcessingEnvironment) : XFiler {
-    override fun write(javaFile: JavaFile) {
+
+    // "aggregating" is ignored in javac, and only applicable in KSP
+    override fun write(javaFile: JavaFile, aggregating: Boolean) {
```

Comment:
forgot to push this change? i saw you made it an enum above but it is still a boolean here?

Reply:
it's pushed, maybe right after you were looking https://github.com/androidx/androidx/pull/155/commits/7697b63bc0325ae4190644dbcd2031f6e23250e6